addons/mail/static/src/components/legacy_emoji_list/legacy_emoji_list.js

```diff
@@ -0,0 +1,57 @@
+/** @odoo-module **/
+
+import { useUpdate } from '@mail/component_hooks/use_update';
+import { registerMessagingComponent } from '@mail/utils/messaging_component';
+import { LegacyComponent } from '@web/legacy/legacy_component';
+
+export class LegacyEmojiList extends LegacyComponent {
```

Comment:
`LegacyComponent` were introduced for components that have legacy features that are no longer available in `Component`, such as `this.trigger`.

Actually I think we could have remove `LegacyComponent`, because the `trigger` are useless since `EmojiList` uses `PopoverView` and not legacy `<Popover>` that had slots. But I prefer to keep it just in case for the legacy, as it's a much safer change.